### ISSUE TITLE
FxA on Firstrun: Remind me later button

### ISSUE THREAD

**Issue by bensternthal** [mozilla/bedrock#5644]
[Northstar idea card](https://projects.growthhackers.com/u/orgs/mozilla-corporation/projects/FUR/cards/FUR-22)

User story: We should offer users the ability to sign-up for a reminder email instead of creating an account, If Visitors aren't ready to commit to creating an account, but want to come back to it later they have an option to do so

**Comment by hoosteeno** [comment]
This is blocked by iframeless.

**Comment by alexgibson** [comment]
Given that this page is moving in-product, I'd say we wontfix this and close the issue/bug?

**Comment by ejregithub** [comment]
Closing as invalid - this page is going in-product
